src/App.jsx

```diff
@@ -11,64 +11,59 @@ import { fetchEbike } from './redux/ebike/ebikeSlice';
 import Signup from './Components/auth/Signup';
 import MyReservations from './Components/reservations/MyReservations';
 import Mainpage from './Components/ebikes/Mainpage';
-// import { fetchReservation } from './redux/reservation/reservationSlice';
 import SpecificBike from './Components/ebikes/SpecificBike';
 import NewReservation from './Components/reservations/NewReservation';
 import { resetMessage } from './redux/auth/authSlice';
 import { fetchLocations } from './redux/location/locationSlice';
 import Message from './Message';
-
-import { fetchReservations, resetReservationMessage } from './redux/reservation/reservationSlice';
+import { resetReservationMessage } from './redux/reservation/reservationSlice';
 import EbikeForm from './Components/ebikes/EbikeForm';
-import { fetchEbikeModels } from './redux/ebike_models/ebikeModelSlice';
-import { resetAddEbikeMessage } from './redux/ebike/addingNewbike';
+import NavigationPanel from './Components/NavigationPanel';
 
 function App() {
   const dispatch = useDispatch();
   const { message, isLogined } = useSelector((state) => state.authSlice);
-  const { message: addReserveMessage } = useSelector((state) => state.reservationSlice);
-  const { addEbikeMessage } = useSelector((state) => state.addingEbikeSlice);
+  const { reservationMessage } = useSelector((state) => state.reservationSlice);
 
   useEffect(() => {
     dispatch(fetchEbike());
-    if (isLogined) dispatch(fetchLocations());
-    if (isLogined) dispatch(fetchReservations());
-    if (isLogined) dispatch(fetchEbikeModels());
-  }, [dispatch, isLogined]);
+    dispatch(fetchLocations());
+  }, [dispatch]);
 
   useEffect(() => {
     setTimeout(() => {
       if (message) dispatch(resetMessage());
-      if (addReserveMessage) dispatch(resetReservationMessage());
-      if (addEbikeMessage) dispatch(resetAddEbikeMessage());
+      if (reservationMessage) dispatch(resetReservationMessage());
     }, 2000);
-  }, [message, dispatch, addReserveMessage, addEbikeMessage]);
+  }, [message, dispatch, reservationMessage]);
 
   return (
     <div className="myApp">
-      { message && <Message message={message} /> }
-      { addReserveMessage && <Message message={addReserveMessage} />}
-      { addEbikeMessage && <Message message={addEbikeMessage} />}
-
       <Router>
+        <NavigationPanel />
+        {message && <Message message={message} />}
+        {reservationMessage && <Message message={reservationMessage} />}
+
         <div className="App">
```

Comment:
Hello Jack, Could you explain why you delete so many configuration having no connection to the Navigation-pane in the App.jsx ?

Reply:
And also the navigation pane does not show at the Reservation Form when I click it.